plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/query/index/HudiColumnStatsIndexSupport.java

```diff
@@ -103,7 +107,7 @@ public Map<String, List<FileSlice>> lookupCandidateFilesInMetadataTable(
                                 .filter(fileSlice -> shouldKeepFileSlice(fileSlice, statsByFileName, regularColumnPredicates, regularColumns))
                                 .collect(Collectors.toList())));
 
-        this.printDebugMessage(candidateFileSlices, inputFileSlices);
+        this.printDebugMessage(candidateFileSlices, inputFileSlices, timer.endTimer());
```

Comment:
@voonhous could you help rebasing #26 once this PR is landed as there will be merge conflicts?

Reply:
Sure!